### PR TITLE
Re-introduce redis

### PR DESCRIPTION
**Pull request by beagles**
Depends-On: https://github.com/openstack-k8s-operators/install_yamls/pull/946

### CODE REVIEW

**Comment by beagles** [comment]
removed "DRAFT" to work on the kuttl tests but still WIP for now

File: tests/kuttl/tests/designate_tls/02-redis-tls-certs.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# hardcode the secret generated by the certificate CR below,
```
Comment (omersch381) [inline tests/kuttl/tests/designate_tls/02-redis-tls-certs.yaml:1]: should we delete those comments?

File: tests/kuttl/tests/designate_tls/05-assert.yaml [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+    - name: designate-public
+  selector:
+    service: designate
+  type: ClusterIP
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+  - script: |
+      if oc get -n $NAMESPACE secret -o json designate-central-config-data | jq '.data["designate.conf"] | @base64d' | sed -e 's/\\n/\n/g' | grep ^backend_url.*redis.*ssl.1;
```
Comment (omersch381) [inline tests/kuttl/tests/designate_tls/05-assert.yaml:242]: I don't think that I understand, on tests/kuttl/tests/basic/02-assert.yaml we have
if ! oc get -n $NAMESPACE secret ...
and on this one we have
if oc get -n $NAMESPACE secret ...

**Review by omersch381 — COMMENTED** [review]
A few questions, but apart from those, it looks good to me.

File: tests/kuttl/tests/designate_tls/05-assert.yaml (hunk 2 above)
Comment (beagles) [reply tests/kuttl/tests/designate_tls/05-assert.yaml:242]: oooo. you're right. Let me check this.

File: tests/kuttl/tests/designate_tls/02-redis-tls-certs.yaml (hunk 1 above)
Comment (beagles) [reply tests/kuttl/tests/designate_tls/02-redis-tls-certs.yaml:1]: I copy pasted it from the redis repo, but perhaps you're right

**Pull request merged**